app/src/main/kotlin/com/wire/android/ui/WireActivityViewModel.kt

```diff
@@ -515,15 +477,47 @@ sealed class CurrentSessionErrorState {
     object SessionExpired : CurrentSessionErrorState()
 }
 
-sealed class NewClientData(open val date: String, open val deviceInfo: UIText) {
-    data class CurrentUser(override val date: String, override val deviceInfo: UIText) : NewClientData(date, deviceInfo)
+sealed class NewClientsDialogData(open val clientsInfo: List<NewClientData>, open val userId: UserId) {
```

Comment:
ViewModels shouldn't care if it's a `Dialog` or not.

It should just worry about the data that is gonna be displayed.

If the Composable that is calling it wants to show it in a list, SnackBar, or system notification, the ViewModel would still be exactly the same.

```suggestion
sealed class NewClientsData(open val clientsInfo: List<NewClientData>, open val userId: UserId) {
```